src/main/java/sirius/db/mixing/BaseEntity.java

```diff
@@ -166,7 +166,10 @@ public void assertUnique(Mapping field,
                                          .map(mapping -> getDescriptor().getProperty(mapping.getName()))
                                          .map(property -> property.getFullLabel()
                                                           + ": "
-                                                          + NLS.toUserString(property.getValue(this)))
+                                                          + (Strings.isEmpty(NLS.toUserString(property.getValueForUserMessage(
```

Comment:
Gehört es nicht einfach in Property.getValueForUserMessage ?

Reply:
Ich verstehe leider nicht was du meinst. =/

Reply:
Meinst du den empty check?

Reply:
Das ist eine gute Idee. Das mache ich so.

Reply:
Done